src/w3c/templates/headers.js

```diff
@@ -3,11 +3,25 @@ import showLogo from "./show-logo";
 import showPeople from "./show-people";
 import showLink from "./show-link";
 
+function decorateSpecTitle(specTitleElem) {
+  specTitleElem.classList.add("title", "p-name");
+  specTitleElem.id = "title";
+}
+
 export default conf => {
   const html = hyperHTML;
+  const specTitleElem = document.querySelector("h1#title, body>h1") || document.createElement("h1");
+  if (specTitleElem.parentElement) {
+    // remove it from the DOM tree, because we are going to relocate it
+    specTitleElem.remove();
+    conf.specTitle = specTitleElem.innerText.trim();
+  } else {
+    specTitleElem.innerText = conf.title;
+  }
+  decorateSpecTitle(specTitleElem);
   return html`<div class='head'>
   ${conf.logos.map(showLogo)}
-  <h1 class='title p-name' id='title'>${conf.title}</h1>
+  ${specTitleElem}
   ${conf.subtitle ? html`
     <h2 id='subtitle'>${conf.subtitle}</h2>
```

Comment:
We should support the same behaviour for `conf.subTitle` too - we just match on "h2#subtitle".